.github/workflows/tests.yml

```diff
@@ -258,7 +258,7 @@ jobs:
           conda-build --version
           pushd .. && git clone https://github.com/conda/conda_build_test_recipe && popd
           call conda create -n blarg -yq --download-only python=2.7||exit 1
-          call conda create -n blarg -yq --download-only python=3.8||exit 1
+          call conda create -n blarg -yq --download-only python=3.10||exit 1
```

Comment:
Why not download a relevant version. Not sure why we need to test conda create here.